advanced_samples/Grafana/README.md

```diff
@@ -0,0 +1,40 @@
+# Grafana Sequential Data Store Sample
+
+**Version:** 1.0.0
+
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/OCS/Grafana_NodeJS?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=1680&branchName=master)
+
+This sample demonstrates how to build a [Grafana](https://grafana.com/) plugin that runs queries against the Sequential Data Store of OSIsoft Cloud Services or Edge Data Store. The sample performs normal "Get Values" calls against a specified stream in SDS, using the time range of the Grafana dashboard. See the [Grafana Documentation](https://grafana.com/docs/grafana/latest/developers/plugins/) for more information on developing Grafana plugins.
+
+## Requirements
+
+- [Grafana 6.4+](https://grafana.com/grafana/download)
+- Web Browser with JavaScript enbaled
+- [NodeJS](https://nodejs.org/en/)
+- If using OSIsoft Cloud Services, register a Client Credentials Client in OSIsoft Cloud Services; a client secret will need to be provided to the sample plugin configuration
+- If using Edge Data Store, the browser must be running local to a running copy of Edge Data Store
+
+## Running the Sample
+
+1. Copy this folder to your Grafana server's plugins directory, like `.../grafana/data/plugins`
+1. (Optional) If using other plugins, rename the folder to `osisoft-cloud-services-sample`
+1. Open a command prompt inside that folder
+1. Install dependencies, using `npm ci`
+1. Build the plugin, using `npm build` (or `npm dev` for browser debugging)
```

Comment:
throws error?

C:\Program Files\GrafanaLabs\grafana\data\plugins\osisoft-cloud-services-sample>npm build
npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?